Sources/KlaviyoCore/AppLifeCycleEvents.swift

```diff
@@ -30,23 +39,21 @@ struct AppLifeCycleEvents {
                     environment.emitDeveloperWarning("failure to start reachability notifier")
                 }
             })
-            .map { _ in KlaviyoAction.start }
+            .map { _ in LifeCycleEvents.forgrounded }
         let backgrounded = environment
             .notificationCenterPublisher(UIApplication.didEnterBackgroundNotification)
             .handleEvents(receiveOutput: { _ in
                 environment.stopReachability()
             })
-            .map { _ in KlaviyoAction.stop }
+            .map { _ in LifeCycleEvents.backgrounded }
         // The below is a bit convoluted since network status can be nil.
         let reachability = environment
             .notificationCenterPublisher(ReachabilityChangedNotification)
-            .compactMap { _ -> KlaviyoAction? in
-                guard let status = environment.reachabilityStatus() else {
-                    return nil
-                }
-                return KlaviyoAction.networkConnectivityChanged(status)
+            .map { _ in
+                // TODO: compact map isn't working, need to fix
```

Comment:
Note to self

Reply:
I was having compiler errors earlier but switched back to compact map and it's working now.